build/config.js

```diff
@@ -11,5 +11,9 @@ module.exports = {
   headerClass: '{{$headerClass}}{{/headerClass}}',
   insideHeader: '{{$insideHeader}}{{/insideHeader}}',
   pageTitle: '{{$pageTitle}}{{/pageTitle}}',
-  propositionHeader: '{{$propositionHeader}}{{/propositionHeader}}'
+  propositionHeader: '{{$propositionHeader}}{{/propositionHeader}}',
+  skipLinkMessage: '{{$skipLinkMessage}}Skip to main content{{/skipLinkMessage}}',
+  globalHeaderText: '{{$globalHeaderText}}GOV.UK{{/globalHeaderText}}',
+  licenceMessage: '{{$licenceMessage}}<p>All content is available under the <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence v3.0</a>, except where otherwise stated</p>{{/licenceMessage}}',
+  crownCopyrightMessage: '{{$crownCopyrightMessage}}© Crown copyright{{/crownCopyrightMessage}}'
```

Comment:
html elements don't seem to work in hogan template values; it comes out as &amp;copy;
In the same way we are using £ instead of &pound; in the currency lamda